### PR TITLE
Updated Global Navigation

### PR DESCRIPTION
**Pull request by Bcrunk**

### CODE REVIEW

File: packages/private-design-output/src/example.html [hunk 1]
```diff
@@ -75,15 +75,14 @@
                 </li>
                 <li><a href="#">Research excellence and impact</a></li>
                 <li><a href="#">Faculties, schools, intitutues & centres</a></li>
-                <li><a href="#">Research facilities</a></li>
+                <li><a href="#">Research facilities and infrastructure and infrastructure</a></li>
```
Comment (uqnm) [inline packages/private-design-output/src/example.html:78]: double up here

File: packages/private-design-output/src/example.html [hunk 2]
```diff
@@ -212,10 +211,9 @@
                           <ul>
                             <li><a href="#">Research excellence and impact</a></li>
                             <li><a href="#">Faculties, schools, intitutues & centres</a></li>
-                            <li><a href="#">Research facilities</a></li>
+                            <li><a href="#">Research facilities and infrastructure and infrastructure</a></li>
```
Comment (uqnm) [inline packages/private-design-output/src/example.html:214]: double up

File: packages/private-design-output/src/homepage.html [hunk 3]
```diff
@@ -85,9 +85,8 @@
...
                             <li><a href="https://research.uq.edu.au/news" data-gtm-label="Research > Research news">Research news</a></li>
                             <li><a href="https://research.uq.edu.au/research-support" data-gtm-label="Research > Research funding, ethics and support">Research funding, ethics and support</a></li>
-                            <li><a href="https://research.uq.edu.au/partner-with-us" data-gtm-label="Research > Industry research collaboration">Industry research collaboration</a></li>
                             <li><a href="https://about.uq.edu.au/faculties-schools-institutes-centres" data-gtm-label="Faculties, schools, institutes and centres">Faculties, schools, institutes and centres</a></li>
-                            <li><a href="https://research.uq.edu.au/research-infrastructure" data-gtm-label="Research > Research facilities">Research facilities</a></li>
+                            <li><a href="https://www.uq.edu.au/research/about/research-infrastructure" data-gtm-label="Research > Research facilities and infrastructure and infrastructure">Research facilities and infrastructure and infrastructure</a></li>
```
Comment (uqnm) [inline packages/private-design-output/src/homepage.html:89]: double up

File: packages/private-design-output/src/homepage.html [hunk 4]
```diff
@@ -272,27 +273,28 @@
                 <ul>
                   <li><a href="https://research.uq.edu.au/news" data-gtm-label="Research > Research news">Research news</a></li>
                   <li><a href="https://research.uq.edu.au/research-support" data-gtm-label="Research > Research funding, ethics and support">Research funding, ethics and support</a></li>
-                  <li><a href="https://research.uq.edu.au/partner-with-us" data-gtm-label="Research > Industry research collaboration">Industry research collaboration</a></li>
                   <li><a href="https://about.uq.edu.au/faculties-schools-institutes-centres" data-gtm-label="Faculties, schools, institutes and centres">Faculties, schools, institutes and centres</a></li>
-                  <li><a href="https://research.uq.edu.au/research-infrastructure" data-gtm-label="Research > Research facilities">Research facilities</a></li>
+                  <li><a href="https://www.uq.edu.au/research/about/research-infrastructure" data-gtm-label="Research > Research facilities and infrastructure and infrastructure">Research facilities and infrastructure and infrastructure</a></li>
```
Comment (uqnm) [inline packages/private-design-output/src/homepage.html:277]: you know the drill

File: packages/storybook-html/stories/components/header/header.html [hunk 5]
```diff
@@ -74,9 +74,8 @@
                       <ul>
                         <li><a href="https://research.uq.edu.au/news" data-gtm-label="Research > Research news">Research news</a></li>
                         <li><a href="https://research.uq.edu.au/research-support" data-gtm-label="Research > Research funding, ethics and support">Research funding, ethics and support</a></li>
-                        <li><a href="https://research.uq.edu.au/partner-with-us" data-gtm-label="Research > Industry research collaboration">Industry research collaboration</a></li>
                         <li><a href="https://about.uq.edu.au/faculties-schools-institutes-centres" data-gtm-label="Faculties, schools, institutes and centres">Faculties, schools, institutes and centres</a></li>
-                        <li><a href="https://research.uq.edu.au/research-infrastructure" data-gtm-label="Research > Research facilities">Research facilities</a></li>
+                        <li><a href="https://www.uq.edu.au/research/about/research-infrastructure" data-gtm-label="Research > Research facilities and infrastructure and infrastructure">Research facilities and infrastructure and infrastructure</a></li>
```
Comment (uqnm) [inline packages/storybook-html/stories/components/header/header.html:78]: "

File: packages/storybook-html/stories/components/header/header.html [hunk 6]
```diff
@@ -261,27 +262,28 @@
             <ul>
               <li><a href="https://research.uq.edu.au/news" data-gtm-label="Research > Research news">Research news</a></li>
               <li><a href="https://research.uq.edu.au/research-support" data-gtm-label="Research > Research funding, ethics and support">Research funding, ethics and support</a></li>
-              <li><a href="https://research.uq.edu.au/partner-with-us" data-gtm-label="Research > Industry research collaboration">Industry research collaboration</a></li>
               <li><a href="https://about.uq.edu.au/faculties-schools-institutes-centres" data-gtm-label="Faculties, schools, institutes and centres">Faculties, schools, institutes and centres</a></li>
-              <li><a href="https://research.uq.edu.au/research-infrastructure" data-gtm-label="Research > Research facilities">Research facilities</a></li>
+              <li><a href="https://www.uq.edu.au/research/about/research-infrastructure" data-gtm-label="Research > Research facilities and infrastructure and infrastructure">Research facilities and infrastructure and infrastructure</a></li>
```
Comment (uqnm) [inline packages/storybook-html/stories/components/header/header.html:266]: guess what

**Review by uqnm — COMMENTED** [review]
ive noted the places where i could see a typo has been introduced - just need to fix that across the files

**Comment by Bcrunk** [comment]
> ive noted the places where i could see a typo has been introduced - just need to fix that across the files

Fixed them up, missed during the find replace

**Review by uqnm — APPROVED** [review]

**Pull request merged**